.github/workflows/_ios-build-test.yml

```diff
@@ -121,15 +114,13 @@ jobs:
         if: inputs.ios-platform == 'SIMULATOR'
         run: |
           # shellcheck disable=SC1091
-          source "${RUNNER_TEMP}/anaconda/bin/activate"
+          source "${MINICONDA_INSTALL_PATH}/bin/activate" "${RUNNER_TEMP}/${CONDA_ENV}"
```

Comment:
Same goes for this line, it can be removed

Reply:
Indeed, thanks! I thought activation was needed in this step as well. The line has been removed.